src/tests/ftest/container/fill_destroy_loop.yaml

```diff
@@ -0,0 +1,25 @@
+hosts:
+  test_servers: 3
+server_config:
+  name: daos_server
+  servers:
+    bdev_class: nvme
+    bdev_list: ["0000:81:00.0"]
+    scm_class: dcpm
+    scm_list: ["/dev/pmem0"]
+timeout: 1200
+pool:
+  name: daos_server
+  control_method: dmg
+  scm_size: 1G
+  nvme_size: 26G
```

Comment:
From this pr (pr-9859/16): I only specified size (without scm_size and nvme_size) and seeing following error:
/usr/bin/dmg -o /etc/daos/daos_control.yml -d -j pool create --group=jenkins --label=TestLabel_1 --size=1G --user=jenkins' finished with 1 after 0.16453170776367188s
2022-08-16 21:37:39,675 dmg_utils        L0542 ERROR| pool create failed: server: code = 604 description = "requested SCM capacity is too small (min 16 MiB per target)"

Reply:
Right. That's because the _**total SCM + NVMe size**_ is 1G. Whereas if you specify `scm_size=1G`, the total SCM size is `1G * num_engines`. If you use `size`, you probably need something like 50G+